scout/server/blueprints/variant/templates/variant/variant.html

```diff
@@ -68,7 +68,7 @@
     </div>
     {% endif %}
 
-    <div class="row">{{ matching_variants(managed_variant, causatives) }}</div>
+    <div class="row"><div class="col-12">{{ matching_variants(managed_variant, causatives) }}</div></div>
```

Comment:
This looked a little sad when there was actually a match - and also a little skewed when no match:
![Screenshot 2021-10-14 at 11 23 40](https://user-images.githubusercontent.com/758570/137311030-17145ec5-32b2-4886-84c3-5bde6354fc66.png)